src/mongoc/mongoc-config.h.in

```diff
@@ -153,5 +153,13 @@
 #  undef MONGOC_NO_AUTOMATIC_GLOBALS
 #endif
 
+/*
+ * Use system crypto profile
+ */
+#define ENABLE_SYSTEM_CRYPTO_PROFILE @ENABLE_SYSTEM_CRYPTO_PROFILE@
+
+#if ENABLE_SYSTEM_CRYPTO_PROFILE != 1
+#  undef ENABLE_SYSTEM_CRYPTO_PROFILE
```

Comment:
Everything needs to be MONGOC_ namespaced.

I think moving it up and rename to MONGOC_ENABLE_CRYPTO_SYSTEM_PROFILE would be nice.